### ISSUE TITLE
Installation (Web Application or API Server)

### ISSUE THREAD

**Issue by gentritabazi01** [laravel/ideas#2273]
When creating a new Laravel project, you can choose between:

An API server
Or, a Web application.

### Web Application
A web application project structure is packed with all the required components to create a fully fledged server render application.

### API Server
On the other hand, the API server is more tailored towards creating JSON API servers and doesn't include all the packages.

**Comment by fefogarcia** [comment]
Would that be different enough from the basic difference between Laravel and Lumen though?

**Comment by lucassmacedo** [comment]
API = Laravel Zero

**Comment by lucassmacedo** [comment]
Laravel API could be a Laravel aplication whitout resources and assets

**Comment by lucassmacedo** [comment]
i like this idea

**Comment by devmsh** [comment]
I tried this before, but end with a conclusion that it didn't make sense.

The simplest problem I face when I tried to disable the blade and session related service providers is that telescope will not work.

I think the separation between web and api alone make more sense, and you will have no benefit from having a lumen like laravel installation.

I have different features that can make the mobile backend development easier and I will share many of them as open source packages soon 👍
